frontend/containers/open-call-page/investor/component.tsx

```diff
@@ -30,60 +31,78 @@ export const OpenCallInvestorAndFooter: FC<OpenCallInvestorProps> = ({ openCall,
+    <>
+      <div className="w-full pt-20 bg-background-middle">
         <LayoutContainer>
-          <h2 className="max-w-2xl mx-auto font-serif text-3xl font-semibold text-white lg:text-4xl">
-            <FormattedMessage defaultMessage="Would you like to apply for this fund?" id="8KmEyC" />
-          </h2>
-          <div className="flex flex-col items-center justify-center gap-4 mt-8 sm:flex-row sm:mt-14">
-            <Button disabled theme="primary-white" onClick={handleApply}>
-              <span className="text-lg">
-                <FormattedMessage defaultMessage="Apply now" id="VR4TEV" />
-              </span>
-            </Button>
-            <span className="text-xl text-white">
-              <FormattedMessage defaultMessage="or" id="Ntjkqd" />
-            </span>
-            <Button disabled={!user} theme="secondary-white" onClick={handleFavoriteClick}>
-              <Icon
-                icon={Heart}
-                className={cx('w-4 mr-3', {
-                  'fill-green-dark': !openCall.favourite,
-                  'fill-white': openCall.favourite,
-                })}
+          <LayoutContainer className="flex flex-col justify-between pb-20 gap-y-12 md:flex-row">
+            <div className="w-full">
+              <h2 className="mb-4 font-serif text-4xl font-bold">
+                <FormattedMessage defaultMessage="Investor" id="nEvNJb" />
+              </h2>
+              <p className="text-gray-700">
+                <FormattedMessage defaultMessage="Who is financing this open call" id="Knhvmr" />
+              </p>
+            </div>
+            <div className="w-full">
+              <ProfileCard
+                className="min-w-full"
+                description={investor?.about}
+                link={`${Paths.Investor}/${investor?.slug}`}
+                name={investor?.name}
+                picture={investor?.picture?.small}
+                profileType="investor"
+                type={investor?.type}
               />
-              <span className="text-lg">
-                <FormattedMessage defaultMessage="Favorite" id="5Hzwqs" />
-              </span>
-            </Button>
-          </div>
+            </div>
+          </LayoutContainer>
         </LayoutContainer>
+        <div className="py-8 text-center sm:py-20 bg-green-dark">
+          <LayoutContainer>
+            <h2 className="max-w-2xl mx-auto font-serif text-3xl font-semibold text-white lg:text-4xl">
+              <FormattedMessage
+                defaultMessage="Would you like to apply for this fund?"
+                id="8KmEyC"
+              />
+            </h2>
+            <div className="flex flex-col items-center justify-center gap-4 mt-8 sm:flex-row sm:mt-14">
+              <Button
+                theme="primary-white"
+                disabled={
+                  !userAccount ||
+                  userAccount.type !== UserRoles.ProjectDeveloper ||
+                  status !== OpenCallStatus.Launched
```

Comment:
Without this change, it's impossible to apply using this button:
```suggestion
                  openCall.status !== OpenCallStatus.Launched
```